roles/openshift_cfme/tasks/main.yml

```diff
@@ -0,0 +1,176 @@
+# PASSED IN ROLE VARIABLES:
+#
+# - template_dir - A remote temp dir to template the PVs in
+
+---
+- name: What are the groups?
+  debug:
+    var: groups.keys()
+
+- name: Where is my template dir
+  debug:
+    var: template_dir
+
+######################################################################
```

Comment:
The remainder of this basically follows the pattern outlined in the [upstream docs](https://github.com/ManageIQ/manageiq-pods).